cmd/helm/flags.go

```diff
@@ -140,6 +143,7 @@ func (p *postRendererString) Type() string {
 }
 
 func (p *postRendererString) Set(val string) error {
+	fmt.Println("set set", val)
```

Comment:
```suggestion
```

Reply:
fail, debugging a different issue and changes got swept in. fixed now